docs/user/user/howto/real-time-events/real-time-events.md

```diff
@@ -14,6 +14,21 @@ Glean ["events" ping](../../pings/events.md) submission can be configured either
 Setting the maximum event threshold to a value of `1` will configure the Glean SDK to submit an "events" ping for each and every [event](../../../reference/metrics/event.md) as they
 are recorded. By default, the Glean SDK will batch 500 events per "events" ping.
 
+As of November 2024, Desktop Release:
+Median user per day: 67 events / 3 pings
+The impact of turning on one event per ping based on the median user would result in an increase of approximately 21 times more event ping volume.
```

Comment:
We should find a better formatting for this. mdbook treats this all as a single line, so at the very least it needs a `.` (dot) after `3 pings` or something (see rest in review comment)

Reply:
I tried to clean it up a bit, see what you think now.